regolith/helpers/u_contacthelper.py

```diff
@@ -82,7 +88,11 @@ def db_updater(self):
             notes = current.get('notes', [])
             aliases = current.get('aka', [])
         else:
-            pdoc.update({"date": dt.today()})
+            if not rc.inst:
```

Comment:
I think you need to raise an exception here, not just a print statement

Reply:
changed to:
`            if not rc.institution:
                raise RuntimeError("Institution is required to create a new contact")`